package.json

```diff
@@ -39,31 +39,49 @@
       }
     }
   },
+  "mocha": {
```

Comment:
To keep the package.json file clean we should extract this config into a separate file called `.mocharc.js`. A typical mocha config file can be created as shown [here](https://github.com/mochajs/mocha/blob/master/example/config/.mocharc.js).